From 6a0310cb73e927839ad13f5f799328cef32a770d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Mantas=20Mikul=C4=97nas?= <grawity@gmail.com>
Date: Wed, 13 Nov 2013 13:36:16 +0200
Subject: [PATCH 196/240] activate: fix crash when -s is passed

getopt_long() was told to accept -s which was never implemented.
---
 src/activate/activate.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/activate/activate.c b/src/activate/activate.c
index fc581b9..a9461bc 100644
--- a/src/activate/activate.c
+++ b/src/activate/activate.c
@@ -346,7 +346,7 @@ static int parse_argv(int argc, char *argv[]) {
         assert(argc >= 0);
         assert(argv);
 
-        while ((c = getopt_long(argc, argv, "+hl:saE:", options, NULL)) >= 0)
+        while ((c = getopt_long(argc, argv, "+hl:aE:", options, NULL)) >= 0)
                 switch(c) {
                 case 'h':
                         help();
-- 
1.8.4.5

